test/valid-data/generic-void/schema.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "MyObject": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "$ref": "#/definitions/MyGeneric<void>"
+                }
+            },
+            "required": [
+                "value"
+            ],
+            "additionalProperties": false
+        },
+        "MyGeneric<number>": {
```

Comment:
Why is this number, not void?

Reply:
.. apparently, because I copy and paste poorly!

Fixing, testing locally!

The test copy pasta raises some interesting questions: perhaps an interesting place to use both [mutation](https://github.com/stryker-mutator/stryker/tree/master/packages/typescript) and [property](https://github.com/dubzzz/fast-check) testing>